src/st/no0/e_slinger.c

```diff
@@ -229,3 +227,24 @@ void EntitySlingerPieces(Entity* self) {
         self->velocityX = -self->velocityX;
     }
 }
+
+void EntitySlingerRib(Entity* self) {
```

Comment:
This was in the next split, have moved it here and renamed with what it represents and a minor tidy up